tutorials/core-concepts/users-and-permissions/managing-accesss-control.md

```diff
@@ -1,5 +1,159 @@
+
+Now you can create new users in the *editor* group, which will have the ability to create new content and manage the content they create, but will lack the great power -- and great responsibility -- of the admin role.
+
+{% hint style='info' %}
+An admin can lock any piece type as `adminOnly` which will remove the ability of users with the `edit` permission (or otherwise customized permissions) to create or edit pieces of that type.
+{% endhint %}
+
+
+## Advanced permissions: creating custom groups, assigning permissions for pieces
+
+So far we've covered two cases:
+
+* The very basic situation where you really only need administrators and guests
+
+* The only slightly less simple situation where you need to add an universal editor group as well.
+
+Next, you'll learn how to create more specific groups with more granular permissions. 
+
+In addition to the default groups, Apostrophe has a convention for permissions that supplies prefixes like `edit-` and `admin-` for modules. Using these you can create "editors" and "administrators" for specific modules and tools without providing any privileges for anything else..
+
+So, for example, if you have an HR coordinator who needs access to upload important documents to an employee portal, but doesn't need any further access, you could provide the `edit-` and `submit-` permission for files to a group with no other permissions.
+
+{% hint style='info' %}
+
+[`apostrophe-worklow`](https://github.com/apostrophecms/apostrophe-workflow) is an optional module which adds workflow features to your site. In order to provide these features, the module modifies aspects of the existing groups, and adds more options for configuring groups. You can learn more about permissions in the [Workflow with permissions section](https://github.com/apostrophecms/apostrophe-workflow#user-content-workflow-with-permissions-limiting-who-can-do-what).
+
+{% endhint %}
+
+### Configuring advanced permissions
+
+To manage these permissions, you'll use the *Groups* menu which is currently hidden from the admin bar. To make the menu option visible, you need to disable `groups` in the `apostrophe-users` module in both `app.js` and in `index.js` for `apostrophe-users`.
+
+1. Open `lib/modules/apostrophe-users/index.js`
+
+2. Remove the `groups [ ]` section.
+
+3. Open `app.js`
+
+4. Remove the `groups [ ]` section from `apostrophe-users` (if it exists).
+
+Now the graphical group management interface is available, and you can create groups, just like any other kind of piece (although you must be an `admin` already to do so). To test this out, let's create the group for HR that we described above:
+
+### Example: Advanced Groups
+
+1. Click the new *Groups* option on the admin bar.
+
+    ![](/.gitbook/assets/user-group-bar.png)
+
+2. Click *Add Group*.
+
+3. Name the group "HR Coordinator".
+
+4. Select the "Permissions" tab on the left.
+
+5. Scroll down and check the boxes for *Edit: File* and *Submit: File*
```

Comment:
I believe there is also an "upload attachment" permission they will need to give out for your example to work. This enables the actual uploads, while "edit file" allows them to create a piece describing that attachment in the "Files" area of the admin bar. The distinction is of course a pain to explain and so it's going to be automatic in 3.0.

You do not need "submit file," which has no function in a typical project right now. Edit File will do.

Reply:
I don't see any permissions for Attachments in the list.